README.md

```diff
@@ -58,26 +58,22 @@ Check out this screencast on how to create and deploy a new Shopify App to Herok
 
 Or if you prefer text instructions the steps in the video are written out [here](https://github.com/Shopify/shopify_app/blob/master/docs/Quickstart.md)
 
-Becoming a Shopify App Developer
---------------------------------
-If you don't have a Shopify Partner account yet head over to http://shopify.com/partners to create one, you'll need it before you can start developing apps.
-
-Once you have a Partner account create a new application to get an Api key and other Api credentials. To create a development application set the Application Callback URL to
+App Tunneling
+-------------
 
-```
-http://localhost:3000/
-```
+In order to install your local app on a shop, use the [App Proxy Controller](#app-proxy-controller-generator) or receive Webhooks from Shopify, your app needs to be on a publicly visible URL. Use a tunneling service like [Forward](https://forwardhq.com/), [RequestBin](https://requestb.in/), [ngrok](https://ngrok.com/) etc. These tools allow you to create a secure tunnel from the public Internet to your local machine.
 
-and the `redirect_uri` to
+For example with ngrok, run this command to set up proxying to Rails' default port:
```

Comment:
can we add a link to ngrok here?

Reply:
maybe also add a note that they need to use the `https` ngrok link, not the http one